nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkServiceImpl.java

```diff
@@ -216,7 +216,9 @@ public boolean await(String commandId, Duration duration) throws InterruptedExce
             }
             Thread.sleep(100);
         } while (deadline > System.currentTimeMillis());
-        log.debug("await timeout for commandId(" + commandId + ") after " + duration.toMillis() + " ms");
+        if (log.isDebugEnabled()) {
+            log.debug("await timeout " + status + " after " + duration.toMillis() + " ms");
```

Comment:
Unless you want to easily backport, you can use log4j2 logger with lambdas now.

Reply:
switch to log4j2 is on my todo list